src/App.js

```diff
@@ -45,6 +47,9 @@ function App() {
           </AppBar>
 
         <Switch>
+            <Route exact path="/">
+              <Redirect to="/search" />
+            </Route>
```

Comment:
Instead of writting full jsx code, create array with content and then map over it to create array of route components